dev/InfoBadge/InfoBadge_themeresources.xaml

```diff
@@ -103,4 +107,76 @@
         </Setter>
     </Style>
 
+    <Style TargetType="local:InfoBadge" x:Key="AttentionDotInfoBadgeStyle" BasedOn="{StaticResource DefaultInfoBadgeStyle}">
+        <Setter Property="Background" Value="{ThemeResource SystemColorHighlightBrush}"/>
```

Comment:
I'm wondering where this backgroung color is coming from? SystemColorHighlightBrush sounds like a high contrast color to me.

Reply:
https://github.com/microsoft/microsoft-ui-xaml/blob/anawish/badge-spec/specs/InfoBadge/images/color-breakdown.PNG

Reply:
Huh. That's odd since there's a SystemFillColorAttentionBrush, which InfoBar uses. I would expect the same here.

Reply:
@anawishnoff can you weigh in?

Reply:
Actually, this is a broader question I want to bring up with design. Should the info state actually use the system's accent color? If the color is yellow/red/green it gives unintentional meaning to that notification. I'm thinking this should always be the 'blue' to have a standard experience across systems.

Reply:
@gabbybilka It should stay consistent with whatever the user chooses as the system's accent colour. It only makes it stand out and distract the user if they choose a different colour other than the default blue.